src/main/resources/citizen/templates/document/countyCourtJudgmentByAdmission.html

```diff
@@ -0,0 +1,226 @@
+<html>
+<head>
+  <meta charset="UTF-8"/>
+  <title>{{ ccj.claim.claim.referenceNumber }}</title>
```

Comment:
`.claim.claim` looks very odd and gives the impression of incorrect code, will suggest to correct this. May be `case.claim` or `claim.claimData`. 

Would require change in `Line 61: ClaimContentProvider` and also places where it's been used.

Reply:
agreed, but perhaps a separate PR as I believe the other CCJ template is also affected